src/module_3/push_noti_explore.md

```diff
@@ -0,0 +1,783 @@
+                (&#x27;logisticregression&#x27;,
+                 LogisticRegression(C=1, penalty=&#x27;l1&#x27;, solver=&#x27;saga&#x27;))])</pre><b>In a Jupyter environment, please rerun this cell to show the HTML representation or trust the notebook. <br />On GitHub, the HTML representation is unable to render, please try loading this page with nbviewer.org.</b></div><div class="sk-container" hidden><div class="sk-item sk-dashed-wrapped"><div class="sk-label-container"><div class="sk-label sk-toggleable"><input class="sk-toggleable__control sk-hidden--visually" id="sk-estimator-id-4" type="checkbox" ><label for="sk-estimator-id-4" class="sk-toggleable__label sk-toggleable__label-arrow">Pipeline</label><div class="sk-toggleable__content"><pre>Pipeline(steps=[(&#x27;standardscaler&#x27;, StandardScaler()),
+                (&#x27;logisticregression&#x27;,
+                 LogisticRegression(C=1, penalty=&#x27;l1&#x27;, solver=&#x27;saga&#x27;))])</pre></div></div></div><div class="sk-serial"><div class="sk-item"><div class="sk-estimator sk-toggleable"><input class="sk-toggleable__control sk-hidden--visually" id="sk-estimator-id-5" type="checkbox" ><label for="sk-estimator-id-5" class="sk-toggleable__label sk-toggleable__label-arrow">StandardScaler</label><div class="sk-toggleable__content"><pre>StandardScaler()</pre></div></div></div><div class="sk-item"><div class="sk-estimator sk-toggleable"><input class="sk-toggleable__control sk-hidden--visually" id="sk-estimator-id-6" type="checkbox" ><label for="sk-estimator-id-6" class="sk-toggleable__label sk-toggleable__label-arrow">LogisticRegression</label><div class="sk-toggleable__content"><pre>LogisticRegression(C=1, penalty=&#x27;l1&#x27;, solver=&#x27;saga&#x27;)</pre></div></div></div></div></div></div></div>
+
+
+
+
+```python
+y_pred = pipe.predict(val_X)
+y_pred = pipe.predict_proba(val_X)[:, 1]
+```
+
+
+```python
+y_pred
+```
+
+
+
+
+    array([0.00915496, 0.01739639, 0.00935807, ..., 0.00344246, 0.00284815,
+           0.00344246])
+
+
+
+
+```python
+precision, recall, _ = precision_recall_curve(val_y, y_pred)
+```
+
+
+```python
+pr_display = PrecisionRecallDisplay(precision=precision, recall=recall)
+pr_display.plot()
+```
+
+
+
+
+    <sklearn.metrics._plot.precision_recall_curve.PrecisionRecallDisplay at 0x2118b771f60>
+
+
+
+
+    
+![png](push_noti_explore_files/push_noti_explore_21_1.png)
+    
+
+
+
+```python
+#TODO fer el mateix per a diferents normas i diferents valors de C, comprovar quin resultat es el millor
+#TODO entendre correctament els resultats, com haurien de sortir i com represetarlos correctament
+```
+
+We've checked that our models execute, now let's test which parameters fit best to out case
```

Comment:
Nice check to start with. I'm missing a non-ML baseline. Always start with a baseline, otherwise you don't know it your ML solution adds any value!